.github/workflows/test.yaml

```diff
@@ -13,10 +13,10 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
```

Comment:
Could maybe add 3.12 too since it's also stable?

Reply:
I excluded it because there was a failure to evaluate dependencies and I didn't want to expand the scope of this PR. This should be addressed in a follow-up

Reply:
done in https://github.com/authzed/authzed-py/pull/102